### PR TITLE
[BACKLOG-38921] Add Stacked-Line Chart

### PR DESCRIPTION
**Pull request by AAravind27**

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 3m 46s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **2**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/cgg-plugin/job/PR-174/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: core/src/main/javascript/pt/webdetails/cgg/resources/define-cfg.js [hunk 1]
```diff
@@ -870,6 +885,8 @@
         "pentaho/ccc/visual/Donut",
         "pentaho/ccc/visual/HeatGrid",
         "pentaho/ccc/visual/Line",
+        "pentaho/ccc/visual/LineAbstract",
+        "pentaho/visual/visual/LineStacked",
```
Comment (dcleao) [inline core/src/main/javascript/pt/webdetails/cgg/resources/define-cfg.js:889]: `pentaho/ccc/visual/LineStacked`

**Review by dcleao — CHANGES_REQUESTED** [review]

**Review by dcleao — APPROVED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 21s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **2**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/cgg-plugin/job/PR-174/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 5m 44s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **2**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/cgg-plugin/job/PR-174/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**